src/js/page/ui/pan-zoom.js

```diff
@@ -1,37 +1,34 @@
-function getXY(obj) {
+const getXY = (obj) => {
```

Comment:
Not against this change, but what's the reasoning?

Reply:
I don't mind top level functions (I think brotli does a good job here), but eh, happy to land this as is to avoid more churn.